tests/subsys/bootloader/external_flash/src/app_update.h

```diff
@@ -0,0 +1,2471 @@
+const unsigned char app_update_bin[] = {
```

Comment:
Add a short comment explaining that the app is/does and what it is used for.

Reply:
The general idea is to have this file generated build time.